### PR TITLE
feat(ReleaseUI): License to SourceFile Mapping

### PR DESCRIPTION
**Pull request by akapti**
[//]: # (Copyright Bosch.IO GmbH 2020)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

> Please provide a summary of your changes here.
> Which issue is this pull request belonging to and how is it solving it? 

* Issue #1305 

> Did you add or update any new dependencies that are required for your change?

* No 

* follow steps mentioned in PR #1188 

> Have you implemented any additional tests?
* No.

### Checklist
Must:
- [x] All related issues are referenced in commit messages and in PR
Signed-off-by: Abdul Kapti <abdul.kapti@siemens-healthineers.com>

### CODE REVIEW

File: frontend/sw360-portlet/src/main/resources/META-INF/resources/html/components/includes/releases/summaryRelease.jspf [hunk 1]
```diff
@@ -76,14 +80,16 @@
 
         <tr>
             <td><liferay-ui:message key="main.licenses" />:</td>
-            <td>
-                <sw360:DisplayCollection value="${release.mainLicenseIds}"/>
+            <td class="actions">
+<%--                 <sw360:DisplayCollection value="${release.mainLicenseIds}"/> --%>
```
Comment (ravi110336) [inline frontend/sw360-portlet/src/main/resources/META-INF/resources/html/components/includes/releases/summaryRelease.jspf:84]: please remove the commented code, same for line 91.

**Review by ravi110336 — COMMENTED** [review]
Code looks good, some minor comments.

**Comment by ravi110336** [comment]
Tested, working fine.
image
image

File: frontend/sw360-portlet/src/main/resources/META-INF/resources/html/components/includes/releases/summaryRelease.jspf (hunk 1 above)
Comment (akapti) [reply frontend/sw360-portlet/src/main/resources/META-INF/resources/html/components/includes/releases/summaryRelease.jspf]: removed the commented line.

Comment (ravi110336) [reply frontend/sw360-portlet/src/main/resources/META-INF/resources/html/components/includes/releases/summaryRelease.jspf]: Code looks good, Please merge it.

**Comment by smrutis1** [comment]
@ravi110336 , Thanks for review and test

**Pull request merged**